codes/1963_cpp_yji0828.cpp

```diff
@@ -0,0 +1,64 @@
+#define _CRT_SECURE_NO_WARNINGS
+#include <iostream>
+#include <cstdio>
+#include <queue>
+#include <string>
+#include <cstring>
+#include <cmath>
+using namespace std;
+queue<int> q;
+int check[10001];
+bool prime[10001];
+void bfs(int start, int dest);
+void eratos();
+
+
+int main() {
+	int T;
+	int start, dest;
+	eratos();
+	scanf("%d", &T);
+	for (int i = 0; i < T; i++) {
+		memset(check, -1, sizeof(check));
+		scanf("%d %d", &start, &dest);
+		bfs(start, dest);
+		if (check[dest] != -1)
+			printf("%d\n", check[dest]);
+		else
+			printf("Impossible\n");
+	}
+
+}
+
+void eratos() {
+	memset(prime, true, sizeof(prime));
+	for (int i = 2; i < sqrt(10000); i++) {
+		for (int j = i + i; j < 10000; j += i)
```

Comment:
i*i에서 부터 보는 게 더 효율적입니다.
```suggestion
		for (int j = i * i; j < 10000; j += i)
```

Reply:
수정했습니다!